packages/vanilla/src/components/global-nav/top-nav/notifications/notifications.scss

```diff
@@ -3,6 +3,7 @@
 .hig__notifications {
   .hig__flyout__panel {
     padding: 0;
+    width: 320px;
```

Comment:
Moved this from `packages/vanilla/src/components/global-nav/top-nav/notifications/_list/_list.scss`. This was to resolve some unintentional x-overflow.